src/nwb_datajoint/common/common_position.py

```diff
@@ -55,17 +58,17 @@ class IntervalPositionInfo(dj.Computed):
     definition = """
     -> IntervalPositionInfoSelection
     ---
-    -> nd.common.AnalysisNwbfile
+    -> AnalysisNwbfile
     head_position_object_id : varchar(40)
     head_orientation_object_id : varchar(40)
     head_velocity_object_id : varchar(40)
     """
 
     def make(self, key):
         print(f'Computing position for: {key}')
-        key['analysis_file_name'] = nd.common.AnalysisNwbfile().create(
+        key['analysis_file_name'] = AnalysisNwbfile().create(
             key['nwb_file_name'])
-        raw_position = (nd.common.RawPosition() &
+        raw_position = (nd.common.common_behav.RawPosition() &
```

Comment:
Do you want to import this first as well? `from .common_behav import RawPosition` at the top and just use `RawPosition` in this line? That way it is easy to see from the top what tables this schema depends on.

Reply:
Okay, should be fixed.